File: programs/Ultrasonic1.java
// Ultrasonic1.java

import ch.aplu.raspi.*;

public class Ultrasonic1
{
  public Ultrasonic1()
  {
    Robot robot = new Robot();
    UltrasonicSensor us = new UltrasonicSensor();
    while (!robot.isEscapeHit())
    {
      double v = us.getValue();
      System.out.println("v = " + v);
    }
    robot.exit();
  }

  public static void main(String[] args)
  {
    new Ultrasonic1();
  }
}

File: scripts/pi2go/Ultrasonic1.py
# Ultrasonic1.py

from raspibrick import *

robot = Robot()
uss = UltrasonicSensor()
n = 0
while not isEscapeHit():
    v = uss.getValue()
    print "n:  ", v
    n += 1
robot.exit()
print "All done"
